versions.tf

```diff
@@ -11,3 +11,7 @@ terraform {
     }
   }
 }
+
+provider "azurerm" {
+  features {}
+}
```

Comment:
```suggestion

```

Provider must not be declared within modules. https://developer.hashicorp.com/terraform/language/modules/develop/providers

Reply:
I see, was using directly from terragrunt.hcl, added config to terragrunts generate block.